win/pkgs/grpc.ps1

```diff
@@ -0,0 +1,70 @@
+#Requires -RunAsAdministrator
+
+Get-Content "$PSScriptRoot/utils/re-entry.ps1" -Raw | Invoke-Expression
+$ErrorActionPreference="Stop"
+
+. "$PSScriptRoot/env/mirror.ps1"
+# . "$PSScriptRoot/env/toolchain.ps1"
```

Comment:
Why comment it out?
Without this ninja won't work (if it works for you that's because I cached it in `setup.ps1`)

Reply:
My mistake